xarray/core/groupby.py

```diff
@@ -66,6 +67,52 @@ def _dummy_copy(xarray_obj):
         raise AssertionError
     return res
 
+def _is_one_or_none(obj):
+    return obj == 1 or obj is None
```

Comment:
But this depends on the interpreter, then. CPython might work, but others might not I guess?